meta/store.go

```diff
@@ -394,6 +397,30 @@ func (s *Store) enableRemoteRaft() error {
 	return s.changeState(rr)
 }
 
+func (s *Store) checkRaftState() error {
```

Comment:
Maybe rename to `enabledLocalRaftIfNecessary`?  Kind of verbose but `checkRaftState`does seem clear based on you explanation.  It will actually enable local raft if it can and it's not already enabled.